src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509chain.c

```diff
@@ -168,7 +168,8 @@ static void MergeStatusCodes(CFTypeRef key, CFTypeRef value, void* context)
         *pStatus |= PAL_X509ChainUntrustedRoot;
     else if (CFEqual(keyString, CFSTR("BasicConstraints")))
         *pStatus |= PAL_X509ChainInvalidBasicConstraints;
-    else if (CFEqual(keyString, CFSTR("UsageConstraints")))
+    else if (CFEqual(keyString, CFSTR("UsageConstraints")) || CFEqual(keyString, CFSTR("BlackListedLeaf")) ||
+             CFEqual(keyString, CFSTR("BlackListedKey")))
```

Comment:
Nit: the line breaks here are a bit strange to me, but I see you're just following the convention already established in this file, so, ok.

Reply:
heh, every time I open this file I always think to myself "I really like the way line wrapping was done here".

Reply:
To each his own :-)